camel/agents/chat_agent.py

```diff
@@ -181,6 +181,12 @@ def __init__(
             context_creator, window_size=message_window_size
         )
 
+        self.input_memory_records = (
```

Comment:
nit: better to begin with underscore for private attributes. so `self._input_memory_records`.

And actually here we don't need assgin it to `self`. Use temp variable and pass it to `init_message` also works

Reply:
updated